src/components/program/program-form/validation.js

```diff
@@ -0,0 +1,10 @@
+import * as Yup from "yup";
+
+export default Yup.object({
+  email: Yup.string()
```

Comment:
add .matches(/^.*(@).*(\.).*$/m, "invalid format for email address") to ensure email address is in the right format

Reply:
We can use directly the method from [Yup](https://github.com/jquense/yup#stringemailmessage-string--function-schema) instead